core/computed/entity-classification.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from './computed-artifact.js';
+import {NetworkRecords} from './network-records.js';
+import {Util} from '../util.cjs';
```

Comment:
can use `util.js`

Reply:
`report/renderer/util.js`? Currently all audits use `core/util.cjs`

Reply:
That's right; disregard, the PR that would do this is still open.

Reply:
Ah, did you mean `UrlUtils.getOrigin? `

Reply:
We have two copies of this function and I realize I added a third 🤦🏽